### PR TITLE
refactor(invoke): Use absolute path and simplify test command

### PR DESCRIPTION
**Pull request by chouetz**
### What does this PR do?
Use absolute paths as we are in a module. Calling the tests is made easier

### Motivation
Technical debt

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided. Except if the `qa/skip-qa` label, with required either `qa/done` or `qa/no-code-change` labels, are applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by brycekahle — APPROVED** [review]

**Review by clarkb7 — APPROVED** [review]

**Review by paulcacheux — APPROVED** [review]

File: tasks/unit_tests.py [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
 @task
 def invoke_unit_tests(ctx):
     """
     Run the unit tests on the invoke tasks
     """
-    for _, _, files in os.walk("tasks/unit-tests/"):
-        for file in files:
-            if file[-3:] == ".py" and file != "__init__.py" and not bool(UNIT_TEST_FILE_FORMAT.search(file[:-3])):
-                ctx.run(f"{sys.executable} -m tasks.unit-tests.{file[:-3]}", env={"GITLAB_TOKEN": "fake_token"})
+    ctx.run(f"{sys.executable} -m unittest discover -s tasks -p '*_tests.py'", env={"GITLAB_TOKEN": "fake_token"})
```
Comment (KevinFairise2) [inline tasks/unit_tests.py:11]: **👏 praise:** ‏
Nice one!

**Review by KevinFairise2 — APPROVED** [review]
LGTM, I also think this is much clearer like that

**Review by pducolin — APPROVED** [review]
Nice !!!

**Review by pgimalac — APPROVED** [review]

**Review by NouemanKHAL — APPROVED** [review]

**Review by ahmed-mez — APPROVED** [review]

**Review by kkhor-datadog — APPROVED** [review]

**Pull request merged**